src/cloudFunction.ts

```diff
@@ -155,7 +161,7 @@ export class CloudFunction {
     envVarList.forEach((envVar) => {
       if (!envVar.includes('=')) {
         throw new TypeError(
-          `Env Vars must be in "KEY1=VALUE1,KEY2=VALUE2" format, received ${envVar}`,
+          `Env Vars and/or Labels must be in "KEY1=VALUE1,KEY2=VALUE2" format, received ${envVar}`,
```

Comment:
Rather than say `Env Vars and/or Labels` can we make it generic like `input must be in "KEY1=VALUE1,KEY2=VALUE2" format, received ${envVar}`, catch any error when we call it, add context as to whether it was for labels or env vars and rethrow it?

Reply:
Hey, thanks for your review. To add a context this way, it will be something like
```typescript
if (opts?.envVars) {
  try {
    envVars = this.parseKVPairs(opts.envVars);
  }
  catch (err) {
    if (err) {
      throw new TypeError(
        `The error was raised during env_vars parse process: ${err.message}`,
      )
    }
  }

  request.environmentVariables = envVars;
}
```
Don't you think that `${envVar}` in the message, will already give enough context for a developer?

Reply:
What do you think if an error raised will be like
```typescript
protected parseKVPairs(values: string): KVPair {
  ...
        throw new TypeError(
          `The expected data format should be "KEY1=VALUE1", got ${pair} while parsing ${values}`,
        );
```
?
Adding both the current token and the whole string may give enough context to identify an issue.

Reply:
Hi @bharathkkb, I've pushed additional changes. As for the context, I think that updating `parseKVPairs` error message text is better. What do you think?

Reply:
@nilcolor that looks great